test/index.js

```diff
@@ -85,7 +85,7 @@ function runMocha(driver, mocha) {
  */
 function browserFilter(browser) {
   return browser.getReleaseName() === 'stable'
-    && ['firefox', 'chrome'].includes(browser.getId());
+    && ['chrome', 'firefox'].includes(browser.getId());
```

Comment:
🤔

Reply:
was just double checking something. selenium-assistant stopped killing firefox 57 properly and I wasn't sure if the order in which the browsers opened/closed had anything to do with it. turns out it doesn't. the ordering doesn't matter and the issue isn't actually a show stopper. it just means if you test locally you need to close firefox when you're done. doesn't seem to bother travis though. I filed https://github.com/GoogleChromeLabs/selenium-assistant/issues/105